masu/processor/aws/aws_report_processor.py

```diff
@@ -121,6 +121,7 @@ def line_item_condition_column(self):
         """Create a property with condition to check for line item inserts."""
         return 'invoice_id'
 
+    # pylint:disable=too-many-locals
```

Comment:
Can you expound on this comment to explain why so many locals are needed?

This is one of pylint's better warnings that is worth listening to and not disabling.

Reply:
I wanted to make the smallest change possible to avoid mucking with critical code. I can break it out into sub-methods.

Reply:
I'm okay with just an extra comment. We don't need to refactor anything right now, but I'd like to avoid just a bare pylint disable without any explanation as to why we've disabled it.